src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowGlobalConfiguration.java

```diff
@@ -19,13 +21,14 @@
 
 import jenkins.model.GlobalConfiguration;
 
-@Extension public class ElectricFlowGlobalConfiguration
+@Extension @Symbol("electricflow") public class ElectricFlowGlobalConfiguration
     extends GlobalConfiguration
 {
 
     //~ Instance fields --------------------------------------------------------
 
-    public List<Configuration> efConfigurations;
+    @Deprecated private transient List<Configuration> efConfigurations;
+    public List<Configuration> configurations;
```

Comment:
Why the new one?

Reply:
or else, the yaml format would be `efConfigurations` and not `configuration`. I felt that `configurations` was easier to understand and write in the configuration. Also, the getter was `getConfigurations` so it could have some weird behaviour with the automagical system of JCasC.

Reply:
I thought the yaml key was `configurations` because the `@DataBoundSetter` on the `setConfigurations` method. Fair enough